### PR TITLE
LINTVS-109 Upgrade to Roslyn 1.0.0

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

File: src/CSharp.CodeAnalysis.DocGenerator/packages.config [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net45" userInstalled="true" />
-  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0-rc2" targetFramework="net452" userInstalled="true" />
-  <package id="Microsoft.CodeAnalysis.Common" version="1.0.0-rc2" targetFramework="net452" userInstalled="true" />
-  <package id="Microsoft.CodeAnalysis.CSharp" version="1.0.0-rc2" targetFramework="net45" userInstalled="true" />
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0" targetFramework="net46" userInstalled="true" />
```
Comment (dbolkensteyn) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: @tvsonar I don't feel good changing the .NET Framework version at the last minute

File: src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml [hunk 2]
```diff
@@ -1,63 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dbolkensteyn) [inline src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml]: @tvsonar this change seems unrelated?

File: src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/TestInput.cs [hunk 3]
```diff
@@ -11,5 +11,6 @@ public int MyMethod()
         }
 
         return 6;
+        ;
```
Comment (dbolkensteyn) [inline src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/TestInput.cs]: @tvsonar this change seems unrelated?

File: src/CSharp.CodeAnalysis.DocGenerator/packages.config (hunk 1 above)
Comment (tvsonar) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: @dbolkensteyn Okay, I reverted back to 4.5 in a separate commit. I'll squash them when we are done.

File: src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml (hunk 2 above)
Comment (tvsonar) [inline src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml]: @dbolkensteyn Yes it is unrelated. I was testing the version change in a different machine, and an issue came up with the tests, so I had to fix it. I won't squash the related commit into the other ones. (Check the different commits, it's in a separate one)

File: src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/TestInput.cs (hunk 3 above)
Comment (tvsonar) [inline src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/TestInput.cs]: @dbolkensteyn same as above. I needed to see if only the configured rules are firing or not, so I added an issue to the code which should not be found by the test.

File: src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml (hunk 2 above)
Comment (dbolkensteyn) [inline src/Tests/SonarQube.CSharp.CodeAnalysis.UnitTest/TestResources/ExpectedOutput.xml]: @tvsonar i dont see the commit in this PR

File: src/CSharp.CodeAnalysis.DocGenerator/packages.config (hunk 1 above)
Comment (dbolkensteyn) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: in fact I'm wondering - why am I still seeing this line ad being added?

**Comment by dbolkensteyn** [comment]
LGTM
